srv/components/controller/files/scripts/provision.sh

```diff
@@ -1153,26 +1160,60 @@ fw_upgrade_health_check()
 # provided in the batchfile as input to the function.
 sftp_cmd_run()
 {
-    _batch_file="$1"
-    _cmd=$(cat $_batch_file)
+    local _batch_file="$1"
+    local _cmd=$(cat $_batch_file)
+    local _scr_sleep_time=30
+    local _scr_max_ntry=10
+    local _scr_ntry=0
+    local _scr_ret=
+    local _sftp_log="$tmpdir/sftp.log"
```

Comment:
what is `tmpdir` value. Do we need to check if dir exists and create one if it doesn't?

Reply:
yes, already handled, it is defined in the main script controller-cli.sh in a following way:
export tmpdir="$logdir/_ctrl_cli_tmp"
[ ! -d $tmpdir ] && mkdir -p $tmpdir